src/task1/map/map_coordinates.txt

```diff
@@ -0,0 +1,2 @@
+top left: (48.521839, -71.653898)
```

Comment:
Can you convert this to valid json (check the trello ticket for the example and just plug and play those numbers)